Extensions/LinkedObjects/linkedobjects.ts

```diff
@@ -7,7 +7,7 @@ namespace gdjs {
    * Manages the links between objects.
    */
   export class LinksManager {
-    private links: { [objectId: number]: gdjs.RuntimeObject[] } = {};
+    private links = new Map<integer, LinkedObjectIterable>();
```

Comment:
Are you sure that this does not break a lot of stuff? The iterator of objects is not the same as for maps, and iterating o er a hash maps children is something used a lot everywhere.